bin/check-required-files

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+# shellcheck source=functions
+source "${BASH_SOURCE%/*}/../sanctuary-scripts/functions"
+
+run_custom_script check-required-files "$@"
+
+license="$(get license-file)"
+declare -i failures=0
+
+check() {
+  if test "$1" "$2" ; then
+    pass "test $1 $2"
+  else
+    fail "test $1 $2"
+    (( failures += 1 ))
+  fi
+}
+
+check -f .circleci/config.yml
```

Comment:
It's unfortunate that this script creates a circleci lock-in. I've tried using sanctuary-script on my own project which uses Travis, and avoiding this check is not that straightforward.

Reply:
Good point! Shall we simply remove this check?

Reply:
That would certainly solve my issue. Perhaps we can also check that all `source-files` are present (not sure if we want to though), and in that spirit, we could add a `dev-files` options or something that defaults to `.circleci/config.yml`. In that case I could avoid this check by adding the following to my config:

```text
dev-files = .travis.yml
```

Reply:
The benefit of such a check is marginal if one must remember to enable the check. In other words, one could just as easily `touch .travis.yml && git add .travis.yml` as add `dev-files = .travis.yml` to __.config__.

I've removed the check above. :)

Reply:
:+1: